core/src/main/java/tech/pegasys/eth2signer/core/service/jsonrpc/exceptions/InvalidDataFormatException.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.eth2signer.core.service.jsonrpc.exceptions;
+
+import com.github.arteam.simplejsonrpc.core.annotation.JsonRpcError;
+
+@JsonRpcError(code = -30001, message = "Invalid data format. Not a hex string")
```

Comment:
Don't think we our own specific error, could this just be INVALID_PARAMS json RPC error?

Reply:
Renamed the exception to use the "Invalid params" standard code.